beetsplug/discogs.py

```diff
@@ -315,30 +339,95 @@ def get_tracks(self, tracklist):
 
         return tracks
 
+    def coalesce_tracks(self, raw_tracklist):
```

Comment:
This is still not working on one example on the initial issue:

https://www.discogs.com/Various-Defqon1-Weekend-Festival/release/7168134

It seems that particular release has the `sub_tracks` defined *inside* the track:
```json
"tracklist" = [
        ...
	{
			"duration": "3:06",
			"position": "",
			"type_": "index",
			"sub_tracks": [
				{
					"extraartists": [...],
					"title": "Eat This",
					"type_": "track",
					"artists": [...],
					"duration": "",
					"position": "2-20A"
				},
				{
					"extraartists": [...],
					"title": "God! What The Hell!",
					"type_": "track",
					"artists": [...],
					"duration": "",
					"position": "2-20B"
				}
			],
			"title": "Eat This / God! What The Hell!"
		},
```

as opposed to the other samples mentioned, where the subtracks are just a regular `track` inside `tracklist`. I'm not sure if the first sample is the exception, or if both are pretty common on discogs?